### ISSUE TITLE
 label:question Any chance you could update to Tensorflow 2.4.x and Keras 2.4.x?

### ISSUE THREAD

**Issue by HourGlss** [Never-github/stock-prediction-with-LSTM-DNN#1]
I'd like to make use of the most current optimizations.

**Comment by Never-github** [comment]
it's an old work so I used the old version of tf. I will update to tf2 and add some new functions in the future. Thanks.

> I'd like to make use of the most current optimizations.
